core/src/test/python/collections_test.py

```diff
@@ -197,5 +197,30 @@ def testDictOrderedDictUpdate(self):
         self.assertEqual('entry1' in dict1 and 'entry2' in dict1, True,
                          "expected ordereddict1 to contain 'entry1' and 'entry2' keys")
 
+    # def testOrderedDictSort(self):
```

Comment:
Why did DS add this?

Reply:
I thought PyOrderedDict had a problem. This test exposed an issue with my own logic, so I asked him to add it permanently. It's a simple to test to sort a dictionary by keys, and add the items to the dictionary in this sorted order. The PyOrderedDict simply cares about the order you add them, but its a good example of how to sort a dictionary with our OrderedDict